Tella/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -377,3 +377,12 @@ This feedback is anonymous, so make sure to include contact information if you w
 "Settings_SettFeedback_ExitFeedback_SheetExpl" = "You can come back anytime to submit your feedback.";
 "Settings_SettFeedback_ExitFeedback_Exit_SheetAction" = "Exit without saving";
 "Settings_SettFeedback_ExitFeedback_Save_SheetAction" = "Save draft";
+
```

Comment:
I think that it's better to add the english strings in all the languages until we have the translated strings

Reply:
Yeah i was going to add it at the end of the code review to avoid merge conflicts if we merge some translations in the meantime.